### DISCUSSION TITLE
Charging: Current (Pilot) - What is this?

### DISCUSSION THREAD

**Discussion by drpau** [teslamate-org/teslamate#2905] [Q&A]
Hi all,
I have recently installed Telsamate and its on and running perfectly fine.

At my home, i have a controlled load circuit which i have the 32amp socket attached to to charge my car.   The controlled load circuit is also powering my hot water system.

For those who don't know what a controlled load circuit is, its a second circuit that the power company provide which is separate to your normal general purpose power outlets.  It only comes on at certain times (10:30pm to 7am) and the cost is significantly reduced from normal power, so there is an obvious advantage of using this to charge the car over night.

Because of my controlled load circuit is powering both hotwater and now charging the car, the electrical meter that controls this is limited to 30amp.

When i charge, i lower my charge setting for the car down to (in this example) 12amp.  But i notice in the graph that there is **Current (Pilot)** which is at 32amps.  

What is Current (Pilot)?  Is it the draw that is coming from socket to UMC, and then the UMC with the car slows down the current to the desired setting?  Therefore its "still trying to draw 32amps" (when I'm limited to 30amp).

As you can see from the graph, after some time, the charging stops and then starts up again. This is repeatable every night.

![Screenshot 2022-09-20 093851](https://user-images.githubusercontent.com/13581275/191137475-54269d51-a355-4bd3-8eaa-fb7a88b11b22.png)

Thank you

**Comment by gacnvahe** [comment]
Hello DrPau.  I was wondering the same exact question, iE---What is CURRENT(PILOT).  I think I may have figured out the answer, but not 100% sure.   CP is a signal on the charging (EVSE) equipment.  It is a signal that in J1772 chargers, that signals the car with a varying Duty Cycle how much current the EVSE(Wall connector) can supply.  My guess is that your wall charger is indicating to the car that it CAN supply up to 32AMPs based on the duty cycle. (And that is what shows up in Tesla Mate). However, your car can also use its on board charger to limit the current.

I was curious what this was because it was showing 48AMP when I was at a DC FAST CHARGER (Supercharger). The current didnt make sense to me, because I was pulling 250kW when the current said 48AMPs.  My guess(Again, GUESS) is that Current (Pilot) doesn't apply to Supercharging because it uses CANBus to do the further communications.
